src/CounterInterceptor/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace TemporalioSamples.CounterInterceptor;
+
+using Temporalio.Client;
+using Temporalio.Worker;
+
+internal class Program
+{
+    private static async Task Main(string[] args)
+    {
+        var client = await TemporalClient.ConnectAsync(
+            options: new("localhost:7233")
+            {
+                Interceptors = new[]
+                {
+                    new SimpleClientCallsInterceptor(),

```

Comment:
You should store this as a field and access the counters off of it instead of making counters static